Gemfile.lock

```diff
@@ -89,6 +89,7 @@ GEM
     database_cleaner (1.6.1)
     descendants_tracker (0.0.4)
       thread_safe (~> 0.3, >= 0.3.1)
+    did_you_mean (1.1.0)
```

Comment:
I think there is a newer version that dependabot will try to update (1.1.2). When that happened it broke the specs in CircleCI v1. Would be good to bump it here and fix any issues with CI.